docs/servo_power_monitor/index.html

```diff
@@ -14,14 +14,20 @@
   </div>
   <div id="controlDiv">
     <a id="downloadAnchorElem" style="display:none"></a>
-    <button id='requestSerialButton'>Start</button>
+    <button id='requestSerialButton'>Start(WebSerial)</button>
+    <button id='request-device'>Start(WebUSB)</button>
     <button id='selectDutSerialButton'>Select DUT Port</button>
     <button id='executeScriptButton'>Execute scripts</button>
     <button id='haltButton'>Stop</button>
     <button id='downloadButton'>Download</button>
     <button id='downloadButton'>Mark</button>
     <button id='analyzeButton'>Analyze displayed range</button>
   </div>
+  <div id="config">
+    <button id="addConfigButton">Add config</button>
```

Comment:
Instead of having "Add Config" and "Delete Config" buttons, it's probably more user friendly to have a single "Add Config" button, and separate "Delete" buttons for each individual config.

Reply:
This way the user can, for example, delete a config in the middle of the list.